pkg/splunk/enterprise/clustermaster.go

```diff
@@ -118,6 +120,9 @@ func ApplyClusterManager(client splcommon.ControllerClient, cr *enterpriseApi.Cl
 		} else {
 			result.Requeue = false
 		}
+		if err != nil {
+			eventPublisher.Warning("Delete", fmt.Sprintf("delete custom resource failed %s", err.Error()))
```

Comment:
can this go to the block in line:119? Also the line 45 can be conditional?

Reply:
what happens when terminating is true but error != nil. I just wanted to be sure its only err != nil

Reply:
for time-being we keep the code as it is.